ui/src/components/Voter/Voter.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Voter.css';
+
+const Voter = (props) => {
+    return (
+        <tr className="poll-voter-row">
+            <td className="poll-voter-id">{props.ind}.</td>
```

Comment:
This way, you extract all the props you need and don't have to constantly type `props.{...}`.